eliot/_action.py

```diff
@@ -106,19 +116,42 @@ def toString(self):
         return "/" + "/".join(map(unicode, self.level))
 
 
-    def nextChild(self):
+    def next_sibling(self):
+        """
+        Return the next L{TaskLevel}, that is a task at the same level as this
+        one, but one after.
+
+        @return: L{TaskLevel} which follows this one.
+        """
+        return TaskLevel(level=self.level.set(-1, self.level[-1] + 1))
+
+
+    def child(self, level=1):
```

Comment:
Or rather, it's not ever used in actual code, only by tests, so better to have tests match the way actual code uses `TaskLevel`?